src/store/state/state-model.ts

```diff
@@ -152,6 +152,7 @@ export const stateModel: StateModelIF = {
     businessNameValid: false,
     expiry: null,
     expiryValid: true,
+    relationships: null,
```

Comment:
Would this be better as an empty array?

Reply:
^^ It depends what you do with it before data is set in it.

Reply:
I think empty array is a the way to go.

Reply:
Well, maybe. It depends. An empty array isn't falsy so ...